content/citation-tags.tsv

```diff
@@ -277,3 +277,18 @@ Zintgraf2017_visualizing	arxiv:1702.04595
 goodfellow2016deep	url:http://www.deeplearningbook.org/
 li2016joint	url:http://www.ijcai.org/Proceedings/16/Papers/403.pdf
 world2004international	url:http://www.who.int/classifications/icd/en/
+ghahramani_protect	arxiv:1707.02476
+uncertainty_types	arxiv:1703.04977
+uncertainty_multi_task	arxiv:1705.07115
+guo_calibration	arxiv:1706.04599
+platt_scaling	url:http://citeseer.ist.psu.edu/viewdoc/summary?doi=10.1.1.41.1639
+out_dist_baseline	arxiv:1610.02136
+temp_out_dist	arxiv:1706.02690
+ai_safety	arxiv:1606.06565
+strong_adversary	arxiv:1705.07263
+retinopathy_uncertainty	doi:10.1038/s41598-017-17876-z
+bayesian_hypernets	arxiv:1710.04759
+mcclure_bayesian	arxiv:1611.01639
+uncertainty_ensembles	arxiv:1612.01474
+domain_adapt_uncertainty	arxiv:1505.07818
```

Comment:
Use the JMLR version `url:http://jmlr.org/papers/v17/15-239.html`?

Reply:
Interesting, a very similar version appears to be https://doi.org/10.1007/978-3-319-58347-1_10

Kinds seems like this study was published twice!